configurations/aws_conf.yaml

```diff
@@ -0,0 +1,14 @@
+run_model_func: pv_site_production.models.psp.psp_model.get_model
+model_path: dev_model.pkl
+
+nwp:
+  cls: pv_site_production.data.nwp_data_sources.NwpDataSource
+  kwargs:
+    path: s3://nowcasting-nwp-development/data/latest.netcdf
+    storage_kwargs:
+      key: ${OCF_AWS_KEY}
```

Comment:
we should need these keys for live. It just has the AWS keys using the IAM role. Locally i use `aws configure` to add my keys. 

Leave it in for this PR, but maybe remove it else where

Reply:
Yeah that works with me.

Reply:
Removing it in this PR!